### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by youhymuk**
- Fixed a typo;
- Added greeting for kottans;

### CODE REVIEW

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Hello! Nice job, but your pull request affects other people's code. This may be due to your editor's auto-formatting. Can you please recheck it? Thanks.

File: README.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ Hi,
 
 This is a training repository for Kottans [frontend](https://github.com/Kottans/frontend),
 [backend](https://github.com/Kottans/backend),
-and [git](https://github.com/Kottans/git-course) courses. 
+and [git](https://github.com/Kottans/git-course) courses.
 You can try sending pull-requests to this repository proposing a change.
 The change can be anything you like, for example: fixing a typo in README.md. Consider this phrase, for example:
 
-*This was an intentionaly mispelld phrase*
+*This was an intentionally misspelled phrase*
```
Comment (miroslav-kolomiets) [inline README.md:13]: Pay special attention to this phrase :)

**Comment by youhymuk** [comment]
Hi! Is it correct now?

**Comment by youhymuk** [comment]
I can't understand what the problem is. I've already resolved the conflict several times...

File: README.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ Hi,
 
 This is a training repository for Kottans [frontend](https://github.com/Kottans/frontend),
 [backend](https://github.com/Kottans/backend),
-and [git](https://github.com/Kottans/git-course) courses. 
+and [git](https://github.com/Kottans/git-course) courses.
```
Comment (miroslav-kolomiets) [inline README.md:9]: There are a few things that need to be improved. There is no need to remove spaces where text is added by other students. This applies to this place and to similar ones below in the document.

File: README.md [hunk 3]
```diff
@@ -1338,9 +1338,29 @@ My name is Yuliia and i'm fond of tigers. They're also cats (even big though :gr
 
 ![Tiger](https://media.giphy.com/media/3gPNRFAf1r1108GsK7/giphy-downsized-large.gif)
 ***
+<<<<<<< HEAD
```
Comment (miroslav-kolomiets) [inline README.md:1341]: This got into the commit due to an incorrectly fixed conflict, you need to remove it. And clean up the code below.

**Comment by miroslav-kolomiets** [comment]
> I can't understand what the problem is. I've already resolved the conflict several times...

This is normal. This is due to the fact that other PRs were merged into the branch.

**Comment by youhymuk** [comment]
I really hope that this time I did everything right)

File: README.md [hunk 4]
```diff
@@ -1339,8 +1339,35 @@ My name is Yuliia and i'm fond of tigers. They're also cats (even big though :gr
...
+***
+## Well, hello there!
+
+![mr Tisha](https://user-images.githubusercontent.com/29441499/95658972-ea26b480-0b26-11eb-9cf7-b1166e609ffe.jpg)
+
+I'm [Vlad](https://github.com/5Mountains/kottans-frontend), Front-end developer who is in the process of formation. And, I believe that taking this course will be an interesting adventure despite the fact that it can be challenging.
+
+
+
+***
```
Comment (miroslav-kolomiets) [inline README.md:1370]:
```suggestion
***

### Hello, kottans:)
My name is [Dima](https://github.com/youhymuk) and I'm glad to study here with you.
I wish you all success!
![kottan](gif/cat-headshot.gif)

***
```

**Comment by miroslav-kolomiets** [comment]
> I really hope that this time I did everything right)

Not yet. You miss this comment https://github.com/kottans/mock-repo/pull/423#discussion_r502816367

In fact, all that should be in the PR is:

```
### Hello, kottans:)
My name is [Dima](https://github.com/youhymuk) and I'm glad to study here with you.
I wish you all success!
![kottan](gif/cat-headshot.gif)
```

 And your file - cat-headshot.gif.

There should be no other changes;)

File: README.md [hunk 5]
```diff
@@ -1239,7 +1239,7 @@ I'm Viacheslav. https://github.com/dviacheslav
 
 
 
-Hello! 
+Hello!
```
Comment (Temu4) [inline README.md:1242]: Hi Dima! 
Your code if good! But you a little bit change previous code in this file (even if it just a `space` or `tab`)
I guess the main reason you did changes like this
Screenshot
is that you use some autoFormat settings in you redactor/IDE. You can try to turn off this behaviour (at least during this task).
P.S. To roll back in repo files history you, for example, can use `git log` and `git checkout commitSHA`

**Review by Temu4 — CHANGES_REQUESTED** [review]

**Comment by youhymuk** [comment]
Hi! Thank you for your review guys! I'm truly hoping that everything is ok at this time...

**Comment by youhymuk** [comment]
But I already see there is the same problem... I don't know what is wrong, because I use default settings in WebStorm

**Comment by youhymuk** [comment]
I used  the hard reset to d648791, before adding my code

**Comment by Temu4** [comment]
@youhymuk the problem is not with your code, but with settings in IDE you use (WebShtorm if I'm right).

Auto-formating is really great tool! 👍  But when you collaborate with other people, and work on the same files, it can add no needed changes to the files.

You can **disable auto-formatting** code (at least for this PR) [--> see this <--](https://stackoverflow.com/questions/2977324/disable-reformatting-code-when-saving-files)

**Comment by youhymuk** [comment]
I can't understand how it works, because I have disabled autoformatting in WS as you said to me, but it was the same problem...And now I did my changes in Sublime Text without any formatting.  How can I see there is a problem only in one place now.

File: README.md [hunk 6]
```diff
@@ -1392,4 +1392,9 @@ I`m *[Hanna](https://github.com/HannaSyn)*, and I am really excited to join comm
 
 ## What's up Kottans!
 My name is [Alex](https:/github.com/usides) and I want to be a dev-cat.
-![Image of DevCat](https://www.nexmo.com/wp-content/uploads/2016/08/dx-jamie-comic.jpg)
\ No newline at end of file
+![Image of DevCat](https://www.nexmo.com/wp-content/uploads/2016/08/dx-jamie-comic.jpg)
```
Comment (nazmariam) [inline README.md:1395]: Please don't make changes in previous code, it'll make conflicts for you and another students.
And also could you check image path? 
Screenshot 2020-10-13 at 15 17 20

**Review by Temu4 — APPROVED** [review]
LGTM! 👍 

P.S.
The image will appear after merge.

**Review by nazmariam — APPROVED** [review]

**Review by nazmariam — APPROVED** [review]
But do you think force push is a good idea? :)

**Comment by youhymuk** [comment]
> But do you think force push is a good idea? :)

Hm, does it have negative consequences on this case?

**Comment by nazmariam** [comment]
> > But do you think force push is a good idea? :)
> 
> Hm, does it have negative consequences on this case?

No, in this case I would not give approval. Just kindly reminder not to abuse force pushes unnecessarily

**Comment by youhymuk** [comment]
> > > But do you think force push is a good idea? :)
> > 
> > 
> > Hm, does it have negative consequences on this case?
> 
> No, in this case I would not give approval. Just kindly reminder not to abuse force pushes unnecessarily

Ok, I'll remember about it, thanks!)

**Review by miroslav-kolomiets — APPROVED** [review]
Now it's good to go, good job!

**Pull request merged**